ops/storage-ddls/cassandra-setup.cql

```diff
@@ -165,9 +165,8 @@ CREATE TABLE computedtiles (
     tiley int,
     externalsourceid text,
     mentioncount bigint,
-    avgsentiment float,
+    avgsentimentnumerator bigint,
```

Comment:
#118 is now merged so if you just rebase this from master the comment will be fixed.